src/hotspot/share/prims/jvmtiImpl.hpp

```diff
@@ -88,7 +88,7 @@ class GrowableCache {
   // (but NOT when cached elements are recomputed).
   void (*_listener_fun)(void *, address*);
 
-  static bool equals(void *, GrowableElement *);
+  static bool equals(GrowableElement *, GrowableElement *);
```

Comment:
Remove spacing between `GrowableElement` and `*`

Reply:
Fixed.